package.json

```diff
@@ -26,13 +26,17 @@
   "author": "Nick Williams",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.0.0 || ^16.0.0-beta || ^16.0.0",
+    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0-beta || ^16.0.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.0"
```

Comment:
Some work is going to have to be done before this PR can be merged. With the addition of a dependency, a bundler is required so that `prop-types` gets included in the build output (currently only babel is used for building, which doesn't do any bundling itself).

I'm happy to do this, but it may be a few days before I get around to it.

Reply:
Actually, I take all this back :) It's not required.

When a user installs react-clipboard, dependencies will be pulled down, and when they bundle, dependencies will be included alongside their application code. All good.

Reply:
👍